code/game/objects/items/weapons/manuals.dm

```diff
@@ -5,6 +5,7 @@
 	icon = 'icons/obj/library.dmi'
 	due_date = 0 // Game time in 1/10th seconds
 	unique = 1   // 0 - Normal book, 1 - Should not be treated as normal book, unable to be copied, unable to be modified
```

Comment:
For whoever asks why dont I just use the `unique` variable, this is used in another places and I am not fully aware of all the implications this may have if I add this onto the scanner, hence the new variable.